liquibase-standard/src/main/java/liquibase/database/core/SQLiteDatabase.java

```diff
@@ -202,7 +203,7 @@ public boolean supportsTablespaces() {
     @Override
     public String getViewDefinition(CatalogAndSchema schema, String viewName) throws DatabaseException {
         String definition = Scope.getCurrentScope().getSingleton(ExecutorService.class).getExecutor("jdbc", this).queryForObject(
-                new RawSqlStatement("SELECT sql FROM sqlite_master WHERE name=" + this.quoteObject(viewName, View.class)),
+                new RawParameterizedSqlStatement(String.format("SELECT sql FROM sqlite_master WHERE name=%s", this.quoteObject(viewName, View.class))),
```

Comment:
This seems like a good candidate for a prepared statement.

Reply:
Looking at this a second time, it's irrelevant, my original unclear point was that the query in this line is still using `String.format` instead of a parameterized query.

Reply:
```suggestion
                new RawParameterizedSqlStatement("SELECT sql FROM sqlite_master WHERE name=?", this.quoteObject(viewName, View.class)),
```

Reply:
update it! 

Thank you.